code/src/App.js

```diff
@@ -1,9 +1,81 @@
-import React from 'react';
+/* eslint-disable no-underscore-dangle */
+import React, { useEffect, useState } from 'react'
+import ThoughtForm from 'components/ThoughtForm'
+import ThoughtItem from 'components/ThoughtItem'
+import Spinner from 'components/Spinner'
+
+const FETCH_API = 'https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/thoughts'
 
 export const App = () => {
+  const [thoughts, setThoughts] = useState([])
+  const [loading, setLoading] = useState(false);
+  const [newThought, setNewThought] = useState('')
+
+  const fetchThoughts = () => {
+    setLoading(true)
+    fetch(FETCH_API)
+      .then((res) => res.json())
+      .then((data) => setThoughts(data))
+      .finally(() => setLoading(false));
+  }
+
+  useEffect(() => {
+    fetchThoughts()
+  }, [])
+
+  const handleFormCleanup = () => {
+    setNewThought('')
+    setLoading(false)
+  }
+  const handleFormSubmit = (event) => {
+    event.preventDefault()
+
+    fetch(FETCH_API, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({ message: newThought })
+    })
+      .then((res) => res.json())
+      .then(() => {
+        fetchThoughts()
+        setNewThought('')
+      })
+  }
+
+  const handleLikesIncrease = (thoughtId) => {
+    fetch(`${FETCH_API}/${thoughtId}/like`, {
+      method: 'POST',
+      headers: {
+        'Content-type': 'application/json'
+      }
+
+    })
+      .then((res) => res.json())
+      .then(() => {
+        fetchThoughts()
+          .finally(() => handleFormCleanup())
```

Comment:
I can't really tell what's blocking the like-function but if I was to pick a guess I would say it's somewhere between line 56-58. Don't take my word for it though!